.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
[OPTIONAL] Even though this is an optional (but highly recommended) change, bear in mind that for **every future project** you will need to set up the linters.

You have done great work setting up GitHub Actions (this file) 💪💪💪. But as part of the good practices and setting up the linters, you must install and configure (add config files) the linters to your project. Please check [this guide on how to set up your HTML/CSS Linters.](https://github.com/microverseinc/linters-config/tree/master/html-css)